app/js/server.js

```diff
@@ -8,10 +8,10 @@
  * DS102: Remove unnecessary code created because of implicit returns
  * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
  */
-const logger = require('logger-sharelatex')
-logger.initialize('chat-sharelatex')
 const metrics = require('metrics-sharelatex')
 metrics.initialize('chat')
+const logger = require('logger-sharelatex')
+logger.initialize('chat-sharelatex')
```

Comment:
was there a reason for the order switch?

Reply:
It is todo with monitoring at google cloud level, metris needs to be started first so it can hook into other libraries which are started up.